.github/workflows/flutter-build.yml

```diff
@@ -0,0 +1,35 @@
+name: Flutter build
```

Comment:
It might be worth it to also add a web build to the CI.

Reply:
Seeing as the web is still in beta and it is already build & published by netlify I don't see a need for doing that here as well.